deploy/examples/operator.yaml

```diff
@@ -498,6 +498,13 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # Enable read affinity for rbd volumes.
+  CSI_ENABLE_READ_AFFINITY: "false"
+  # CRUSH location labels define which node labels to use
+  # as CRUSH location. This corresponds to the key values used in
+  # ceph CRUSH map
```

Comment:
Added a link to https://rook.io/docs/rook/v1.10/CRDs/Cluster/ceph-cluster-crd/#osd-topology .
Chose a released version since it less likely to change compared to latest, 
let me know you have any other way to point users to this section.